pkg/controller/controller.go

```diff
@@ -144,7 +144,17 @@ func (c *Controller) sync(key string) error {
 	desiredCRs := c.rbacProvider.ConvertAthenzModelIntoIstioRbac(domainRBAC)
 	currentCRs := c.rbacProvider.GetCurrentIstioRbac(domainRBAC, c.configStoreCache)
 
-	changeList := c.computeChangeList(currentCRs, desiredCRs)
+	errHandler := func(err error, item *processor.Item) error {
```

Comment:
Should we move this outside in order to avoid allocating a new variable every sync call?